Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/connected_components.h

```diff
@@ -456,9 +458,10 @@ std::size_t keep_largest_connected_components(PolygonMesh& pmesh,
  * the size of a connected component is the number of faces it contains), but it is also possible
  * to pass custom sizes, such as the area of the face.
  *
- * Property maps for `CGAL::face_index_t` and `CGAL::vertex_index_t`
- * must be either available as internal property maps
- * to `pmesh` or provided as \ref pmp_namedparameters "Named Parameters".
+* If `PolygonMesh`
+*  has a non writable internal property map
+*  for `CGAL::face_index_t` or `CGAL::vertex_index_t` and no `face_index_map` (respectively `vertex_index_map`) is given
+*  as a named parameter, then the internal one(s) must be initialized. Otherwise, it will be.
```

Comment:
```suggestion
*  for `CGAL::face_index_t` (resp. `CGAL::vertex_index_t`) and no `face_index_map` (resp. `vertex_index_map`) is given
*  as a named parameter, then the internal one must be initialized. Otherwise, it will be.
```